### PR TITLE
Redesign theme

### PR DESCRIPTION
**Pull request by mjhouse**
### Overview

Changed a lot of css and restructured, removed or otherwise modified the structure of the site. The colors are based around [this palette](https://coolors.co/1a6187) and shades that go darker and lighter. I also removed some elements from the settings page etc. and a full breakdown of what was changed and why is in issue #48. The most visible changes are to the banner image, post listing and comment pages:

image

And on mobile it looks like this:

image

### Changes

* Moved title of posts in post listing above username and time
* Moved time ahead of username (`8m ago by <name>` instead of `<name> 8m ago`)
* Removed `stretched-link` class that caused title link to override username/usernote links (clicking anywhere on a post list element went to the post)
* Commented out the "Mark distinguished" and "Mark club" options on post listings
* Commented out the "Copy link" and "Votes" links on post elements. Link can be copied by visiting the post and taking it from the url bar, votes is unnecessary since you can see them on the left anyway.
* Replaced `default_thumb_link.webp`
* Replaced `default_text.webp`
* Removed GIF button in comments.html and submission.html
* Added a check to get rid of an error in `marked.custom.js`:196
* Restructured the placeholder comment message html (`submission.html`:562) and added some code to remove it (`comments_v.js`:246) when a reply is submitted (because the comment section isn't empty anymore)
* Vertically centered username text in dropdown menu at top right
* Removed Donate button from dropdown menu at top right
* Changed color of collapse sidebar
* Removed store and chat icons on mobile view of page
* Changed all "Simps"/"Haters" stuff to "Upvoters"/"Downvoters" 
* Reworked contact form a bit
* Removed Profile Anthem section from settings page
* Removed Name Color section from settings page
* Removed Enemies section from settings page
* Removed PayPig button from Settings > Security > Email

### CODE REVIEW

File: files/assets/js/marked.custom.js [hunk 1]
```diff
@@ -188,13 +188,14 @@ function safeMarkdown(input) {
...
-	
+	var input = document.getElementById(first);
 	var dest = document.getElementById(second);
-	for (var i = 0; i < dest.children.length; i++) {
-		dest.removeChild(dest.children[i]);
+	if(dest && input && input.value.trim() !== ''){
+		for (var i = 0; i < dest.children.length; i++) {
+			dest.removeChild(dest.children[i]);
+		}
+		dest.appendChild(safeMarkdown(input.value));
```
Comment (JulianRota) [inline files/assets/js/marked.custom.js:197]: image

Something is causing the reformatting of the preview to go wrong. I'm assuming it's this stuff, but not 100%.

Comment (mjhouse) [reply files/assets/js/marked.custom.js:197]: According to TwoLargeSnakesMating on discord, this issue is found upstream in the rDrama codebase as well. I'm going to make an issue for it.

Comment (mjhouse) [reply files/assets/js/marked.custom.js:197]: The new issue is #105. I'm going to resolve this because I don't think it's related to this request.

Comment (JulianRota) [reply files/assets/js/marked.custom.js:197]: No problem, it doesn't seem like it's really a deal-breaker anyways for the site to function at all

**Review by JulianRota — APPROVED** [review]

**Pull request merged**